Unable install 1.0.3

### Reproduction
https://stackblitz.com/edit/nuxt-starter-zbwbavuy?file=package.json

run "npm i" in console

### Describe the bug
After the update to version 1.0.3, am unable to install nitro-applicationinsights. 
Im getting this error: 

`npm error code EUNSUPPORTEDPROTOCOL
npm warn ERESOLVE overriding peer dependency
npm error Unsupported URL Type "link:": link:./`

When i compare v1.0.2 with 1.0.3 i seems these 2 lines have been added i package.json:

77 : "nitro-applicationinsights": "link:./",

This also breaks the installation of nuxt-applicationinsights

### Additional context

_No response_

### Logs

_No response_

--- a/package.json
+++ b/package.json
@@ -73,8 +73,7 @@
     "mlly": "^1.7.4",
     "nitro-opentelemetry": "^0.7.4",
     "nitro-test-utils": "^0.9.1",
-    "pathe": "^2.0.3",
-    "nitro-applicationinsights": "link:./"
+    "pathe": "^2.0.3"
   },
   "peerDependencies": {
     "h3": ">=1.15.0",

